packages/aws-cdk/lib/api/deployments/cloudformation.ts

```diff
@@ -499,10 +507,15 @@ function toCfnTags(tags: { [id: string]: string }): Tag[] {
   }));
 }
 
-export async function cleanupOldChangeset(changeSetName: string, stackName: string, cfn: ICloudFormationClient) {
+async function cleanupOldChangeset(
```

Comment:
does `cleanupOldChangeset` not get exposed to users of `aws-cdk`? asking if you've checked that it's safe to unexport this function

Reply:
It's only use is inside this file. We have previously compiled a list of symbols that are "externally used" and this is not part of it.